### PR TITLE
Added Tree algos

### PR DESCRIPTION
**Pull request by imshubham27**

### CODE REVIEW

File: python/Tree/BTreefrompost&in.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Node:
```
Comment (manimaran96) [inline python/Tree/BTreefrompost&in.py:1]: @imshubham27 , Add documentation for this program.

File: python/Tree/Generic_Tree.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
+class Node:
```
Comment (manimaran96) [inline python/Tree/Generic_Tree.py:1]: @imshubham27 , Add documentation for this program(Generic_Tree).

File: python/Tree/BTreefrompost&in.py (hunk 1 above)
Comment (imshubham27) [reply python/Tree/BTreefrompost&in.py:7]: Yes I have added. Please review again

File: python/Tree/Generic_Tree.py (hunk 2 above)
Comment (imshubham27) [reply python/Tree/Generic_Tree.py:12]: Yes I have added. Please review again

**Pull request merged**